src/server.tsx

```diff
@@ -30,6 +30,12 @@ app.use(express.json({ limit: '50mb' }));
 app.use(cors());
 app.options('*', cors());
 
+// Logging
+app.use((req: express.Request, res: express.Response, next: express.NextFunction) => {
+    next();
```

Comment:
I was surprised at having to call `next()` first in the middleware! This was the only incantation I found that worked correctly with the error handling middleware (i.e. logged the correct HTTP code).

Reply:
Very odd. I tried this with the `next()` function called before and after the `console.log` and saw no difference - whether there is a JSON validation error or not 🤔 Going to merge as is but probably worth digging a bit deeper on this one to find out why you saw something else here.